### ISSUE TITLE
Previous icon and reset icons overlap

### ISSUE THREAD

**Issue by apinsard** [wakirin/Litepicker#86]
**Describe the bug**
![Screenshot_2020-05-15 Locations courte durée Rennes – Tourisme ou Affaires – Cocoonr](https://user-images.githubusercontent.com/1536035/82078535-8ae08780-96e1-11ea-985c-aa3810162638.png)

**To Reproduce**
```js
      const options = {
        element: this,
        format: 'D MMM YYYY',
        lang: 'fr-FR',
        numberOfMonths: 2,
        numberOfColumns: 2,
        minDate: new Date(),
        selectForward: false,
        selectBackward: false,
        splitView: false,
        singleMode: false,
        autoApply: false,
        showTooltip: true,
        mobileFriendly: true,
        hotelMode: true,
        autoRefresh: true,
        moveByOneMonth: true,
        useResetBtn: true,
        tooltipText: {
          one: 'nuit',
          other: 'nuits',
        }
      };
      if ($(this).data('end-input')) {
        options.elementEnd = $($(this).data('end-input'))[0];
      }
      new Litepicker(options);
```

**Expected behavior**
Previous icon and reset icon should be spaced. (Also have to consider that icon size may vary according to `buttonText` option.

**Comment by scott-fisher-iii** [comment]
I was seeing this as well and opted to just not use the `useResetBtn` option

**Comment by wakirin** [comment]
Fixed in v1.4.4
